ale_linters/ansible/ansible_lint.vim

```diff
@@ -18,28 +18,16 @@ function! ale_linters#ansible#ansible_lint#Handle(buffer, lines) abort
         endif
     endfor
 
-    " Matches patterns line the following:
-    "
-    " test.yml:35: [EANSIBLE0002] Trailing whitespace
-    let l:pattern = '\v^([a-zA-Z]?:?[^:]+):(\d+):?(\d+)?: \[?([[:alnum:]]+)\]? (.*)$'
+    let l:pattern = '\v^([^:]+):(\d+):%((\d+):)? ([-[:alnum:]]+) (.*)$'
     let l:output = []
 
     for l:match in ale#util#GetMatches(a:lines, l:pattern)
-        let l:code = l:match[4]
-
-        if l:code is# 'EANSIBLE0002'
-        \&& !ale#Var(a:buffer, 'warn_about_trailing_whitespace')
-            " Skip warnings for trailing whitespace if the option is off.
-            continue
-        endif
```

Comment:
Keep support for ignoring warnings about trailing whitespace. The ALE option is there for users who have already configured Vim to automatically remove trailing whitespace, so they can edit code without seeing problems they'll fix automatically anyway.

Reply:
With option -x yaml no reporting of trailing whitespace will be done. It's responsibility of yamllint linter. Which is a separate definition.

Reply:
Agree with this. Having both yamllint and ansible-lint reporting same thing is unnecessary.